app/component/IndexPage.js

```diff
@@ -136,30 +135,63 @@ class IndexPage extends React.Component {
     }
   }
 
-  openFavourites = () => {
-    this.replace('/suosikit');
+  push = (path) => {
```

Comment:
The replace param? I am not sure. Anyway, if you switch between the tabs a couple of times in mobile,  without the additional logic you will have  to click equally many times to get back to the front page.  Now back takes you there immediately. That feels convenient but I may be wrong.